nh_clinical/patient.py

```diff
@@ -84,6 +84,16 @@ def _get_name(self, cr, uid, ids, fn, args, context=None):
             result[r['id']] = self._get_fullname(r)
         return result
 
+    def name_get(self, cr, uid, ids, context=None):
```

Comment:
Ah I see you're actually overriding the method there, to be fair that should have been obvious because it's model. However one thing I like in Java is that there's an `@Override` annotation you can use to be explicit about overriding. There's unfortunately no equivalent in Python :crying_cat_face: